viz/charts/linkerd-viz/templates/metrics-api-policy.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: policy.linkerd.io/v1alpha1
+kind: Server
+metadata:
+  namespace: {{.Values.namespace}}
+  name: metrics-api
+  labels:
+    linkerd.io/extension: viz
+    component: metrics-api
+  annotations:
+    {{ include "partials.annotations.created-by" . }}
+spec:
+  podSelector:
+    matchLabels:
+      linkerd.io/extension: viz
+      component: metrics-api
+  port: http
+  proxyProtocol: gRPC
```

Comment:
I believe that the metrics-api uses a weird gRPC over HTTP protocol.  But from the proxy's point of view, it should just look like HTTP/1.1

Reply:
:+1: